CHANGELOG.md

```diff
@@ -25,6 +25,10 @@
     room keys.
     ([#85](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/85))
 
+-   Add `OlmMachine.registerDevicesUpdatedCallback` to notify when devices have
+    been update.
+    ([#88](https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/pull/88))
+
```

Comment:
This won't be part of v4.0.0. Please could you move it to an "UNRELEASED" section